nodejs/helpers.js

```diff
@@ -0,0 +1,172 @@
+const fs = require("fs")
+const mongodb = require("mongodb")
+const { ClientEncryption } = require("mongodb-client-encryption")(mongodb)
```

Comment:
you'll need to change this to use the "non-injected" approach. We just released `v1.0.0-rc0` which uses that:
```
const { ClientEncryption } = require("mongodb-client-encryption");
```

Reply:
Ah, can do. I'll update dependencies as well.